examples/experimental/dagster-airlift/README.md

```diff
@@ -101,63 +114,96 @@ defs = Definitions.merge(
 )
 ```
 
+</details>
+
 ## Observing Assets
 
-The next step is to observe the assets that are orchestrated from Airflow. In order to do that we must create the corresponding definitions in the Dagster deployment.
+The next step is to observe data assets that are orchestrated from Airflow. In order to do this, we must define the relevant assets in the Dagster code location.
 
-We have an included example at `examples/experimental/dagster-airlift/examples/dbt-example`. We suggest mimicking the structure of this project as a starting point.
+To add definitions corresponding to Airflow tasks, you need to use the `orchestrated_defs` argument to `build_defs_from_airflow_instance`.
 
-To add definitions that an Airlift-enabled deployment will observed, you need to use the `orchestrated_defs` argument to `build_defs_from_airflow_instance`
+(TODO: New observation logic)
 
-_Note: This also accepts an object of type `Definitions`. We have recently added a function, `Definitions.merge`, which allows users to combine and compose `Definitions` objects, which this utilizes._
+In our example, we have three seqeuential tasks:
 
-For most projects we anticipate the following process:
+1. `load_raw_customers` loads a CSV file of raw customer data into duckdb.
+2. `run_dbt_model` builds a series of dbt models (from [jaffle shop](https://github.com/dbt-labs/jaffle_shop_duckdb)) combining customer, order, and payment data.
+3. `export_customers` exports a CSV representation of the final customer file from duckdb to disk.
 
-- Create a new factory class for every operator type or specialized use case of an operator (e.g. a BashOperator invoking a dbt project).
-- Use that factory to create definitions associating with each Airflow task.
+The first and third tasks involve a single table each. We can manually construct `AssetSpec`s that match the assets which they build.
 
-In our example we have a pre-written factory class (in `examples/experimental/dagster-airlift/dagster_airlift/dbt/multi_asset.py` and installable by `uv pip install dagster-airlift[dbt]`) that is associated with invoking a dbt project:
+To build assets for our dbt invocation, we can use a Dagster-supplied factory (in `examples/experimental/dagster-airlift/dagster_airlift/dbt/multi_asset.py` and installable by `uv pip install dagster-airlift[dbt]`).
 
-```python
-from dagster_airlift.def_factory import DefsFactory
+### Mapping assets to tasks
 
-@dataclass
-class DbtProjectDefs(DefsFactory):
-    dbt_project_path: Path
-    name: str
-    group: Optional[str] = None
+The `from_task` and `from_dag` utilities link created assets to their underlying tasks. Assets which are properly linked will be materialized by the Airlift sensor once the corresponding task completes.
```

Comment:
https://github.com/dagster-io/dagster/pull/23848

'dag_defs' and `task_defs` current proposal